support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -194,15 +193,6 @@ const ContributionThankYou = ({
     component: <ContributionThankYouSignIn email={email} csrf={csrf} />,
     shouldShow: isKnownEmail && !user.isSignedIn,
   };
-  const marketingConsentAction = {
-    component: (
-      <ContributionThankYouMarketingConsent
-        email={email}
-        csrf={csrf}
-      />
-    ),
-    shouldShow: true,
-  };
```

Comment:
I think we might want to show this for one offs?

Reply:
👍